sunpy/timeseries/tests/test_timeseriesbase.py

```diff
@@ -574,14 +503,6 @@ def test_concatenation_different_data_ts_error(eve_test_ts, fermi_gbm_test_ts):
         eve_test_ts.concatenate(fermi_gbm_test_ts, same_source=True)
 
 
-def test_concatenation_different_data_list_error(eve_test_ts, fermi_gbm_test_ts):
-    # Take two different data sources, pass one as an iterable and concatenate
-    # but set with the same_source kwarg as true. This should not concatenate.
-    with pytest.raises(TypeError, match="TimeSeries classes must match if "
-                                        "'same_source' is specified."):
-        eve_test_ts.concatenate([fermi_gbm_test_ts], same_source=True)
-
-
```

Comment:
This removes coverage for [this line in `sunpy/timeseries/timeseriesbase.py`](https://app.codecov.io/gh/sunpy/sunpy/compare/6078/changes#D4L480) although probably fine